doc/releases/changelog-dev.md

```diff
@@ -10,6 +10,12 @@
 
 <h3>Deprecations 👋</h3>
 
+* The old return type and associated functions ``qml.enable_return`` and ``qml.disable_return`` are removed.
+ [#4503](https://github.com/PennyLaneAI/pennylane/pull/4503)
+
+* The ``mode`` keyword argument in ``QNode`` is removed. Please use ``grad_on_execution`` instead.
+ [#4503](https://github.com/PennyLaneAI/pennylane/pull/4503)
+
```

Comment:
Since this the the completion of a deprecation, is it now a "breaking change"?

Reply:
Yes I move it